game-core/src/main/java/games/strategy/triplea/ai/pro/AbstractProAi.java

```diff
@@ -86,24 +90,35 @@ public AbstractProAi(
     storedPurchaseTerritories = null;
     storedPoliticalActions = null;
     storedStrafingTerritories = new ArrayList<>();
+
+    // in case there are multiple Pro AIs running at the same time, unregister the one
+    // created before creating one so that there is only one menu item for all of the ProAI
+    DebugMenu.unregisterMenuCallback("Hard AI");
+    DebugMenu.registerMenuCallback("Hard AI", this::initialize);
```

Comment:
The "Hard AI" label is in multiple places, it looks like it is a single thing and we would want to keep it consistent. I think this is a place where extracting it to a constant would be a good move so that the linkage between this label and what players select is explicit and would be the same value.